src/components/Spinner.js

```diff
@@ -2,7 +2,8 @@ import classnames from 'classnames';
 
 // @ts-ignore
 import spinnerSVG from '../../images/icons/spinner--spokes.svg';
-import { registerIcon, SvgIcon } from './SvgIcon';
+import { Icon } from './Icon';
+import { registerIcon } from './SvgIcon';
```

Comment:
Moving `registerIcon` into `Icon` is on my short list and is tracked in https://github.com/hypothesis/frontend-shared/issues/224